src/templates/keyboardShortcut.ts

```diff
@@ -0,0 +1,29 @@
+import { html, HTMLTemplateResult } from "lit";
+import { when } from "lit/directives/when.js";
+import { loop } from "../helpers/directives";
+
+export interface KeyboardShortcut {
+  keys: string[];
+  description?: string;
+  hasMouse?: boolean;
+}
+
+/**
+ * @description
+ * A standardized template to display keyboard shortcuts and shortcut
+ * combinations
+ */
+export function keyboardShortcutTemplate(shortcut: KeyboardShortcut): HTMLTemplateResult {
+  const isAscii = shortcut.keys.every((key) => key.charCodeAt(0) < 128);
+  const hasUpperCase = shortcut.keys.some((key: string) => key === key.toUpperCase());
+  const shouldShowShift = hasUpperCase && isAscii;
```

Comment:
perhaps we can use a shift symbol (`shift = Symbol("shift")`) in our key combinations so we don't have to rely on this heuristic - which is easy to make a mistake with. 

Or we could look for `⇧` in the key combos - they're a dev interface afterall

Reply:
I think I like the idea of using the ⇧ character

Maybe some sort of constant so that if we want to change the appearance of the shift character we can just change the constant

e.g.

```ts
const shiftCharacter = "⇧" as const;
```